### PR TITLE
added a changelog

### PR DESCRIPTION
**Pull request by Lectem**
To be updated as things are added to the library

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  - TitleList Renamed to AM_TitleEntry (3ds/services/am.h)
+  -  Added the missing struct and functions for Y2R (3ds/services/y2r.h)
+
+* Fixes :
+
+  - a35abcb APT fix status event issue
+  - 6bdfa1a aptInit Wait for APT_RUNNING
+  - 0e7755a Fix GPUCMD_AddSingleParam() C++ compatibility issue 	
+
+## version 0 to 6.0
```
Comment (smealum) [inline CHANGELOG.md]: 6.0 ?

Comment (Lectem) [inline CHANGELOG.md]: whoops, forgot the 0.

**Comment by Lectem** [comment]
Added " - Some service prefixes were changed (eg. Y2R -> Y2RU)"

**Comment by fincs** [comment]
I plan on commiting a changelog based on this one once great-refactor is merged into master.

**Pull request closed without merging**